File: year2020/month11/day1119/ms_1119.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string data[50];
	string word = "";
	int i = -1;

	while (word != "0" && i < 50) {
		getline(cin, word);

		if (word.length() <= 100) {
			i++;
			data[i] = word;
		}
	}

	cout << i << endl;

	for (int j = 0; j < i; j++) {
		if (j % 2 == 0) {
			cout << data[j] << endl;
		}
	}

	return 0;
}

File: year2020/month11/day1119/ms_1119.cs
using System;

namespace algorithm_104
{
    class ms_1119
    {
        static void Main(string[] args)
        {
            String[] data = new String[50];
            String word = "";
            int i = -1;

            while(word != "0" && i < 50)
            {
                word = Console.ReadLine();

                if(word.Length <= 100)
                {
                    i++;
                    data[i] = word;
                }
            }

            Console.WriteLine(i);

            for(int j = 0; j < i; j++)
            {
                if(j%2 == 0)
                {
                    Console.WriteLine(data[j]);
                }
            }
        }
    }
}
